src/main/java/com/couchbase/lite/support/Batcher.java

```diff
@@ -55,7 +65,7 @@ public Batcher(ScheduledExecutorService workExecutor,
         this.processor = processor;
         this.inbox = new ArrayList<T>();
```

Comment:
From line 66 - 68, variables are initialized with their declaration. We could remove line 66 - 68. And inbox could be initialized with its declaration.